# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mc-mp/intr_flag.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

*** mc-4.1.40-pre9/src/view.c.orig	2004-08-28 08:09:16.000000000 +0200
--- mc-4.1.40-pre9/src/view.c	2009-08-06 15:45:49.000000000 +0200
***************
*** 639,650 ****
  		if (view->data != (unsigned char*)0)
  			free(view->data);
  		close_view_file (view);
- #ifndef __BEOS__
- 	if (intr_flag) {
- 	    view->partial = 1;
- 	    view->s.st_size = 0;
- 	}
- #endif /*__BEOS__*/
  		return init_growing_view (view, 0, filename);
      }
      view->first = 0;
--- 639,644 ----
